src/platforms/apple/common/dsym.mdx

```diff
@@ -179,7 +179,7 @@ if which sentry-cli >/dev/null; then
 export SENTRY_ORG=___ORG_SLUG___
 export SENTRY_PROJECT=___PROJECT_SLUG___
 export SENTRY_AUTH_TOKEN=YOUR_AUTH_TOKEN
-ERROR=$(sentry-cli upload-dif "$DWARF_DSYM_FOLDER_PATH" 2>&1 >/dev/null)
+ERROR=$(sentry-cli upload-dif --force-foreground "$DWARF_DSYM_FOLDER_PATH" 2>&1 >/dev/null)
```

Comment:
We currently have an issue on RN as well, See https://docs.sentry.io/platforms/react-native/troubleshooting/#ios-build-script-failed
@jan-auer was supposed to investigate that on sentry-cli, @jan-auer do you have any updates on that?

Reply:
Thanks for the info. I think it even makes sense to add this flag if we fix the bug as Xcode and then wait until the upload is finished.

Reply:
That has the side effect of slowing down every build release, that's why it's not the default behavior.
Think about slow connection and large apps that spit out hundreds of MBs of symbols.

Reply:
Better to mention below then?